### PR TITLE
SM-784: Enable Project Create/Edit/Delete CLI Commands

### PR DESCRIPTION
**Pull request by coltonhurst**
## Type of change

```
- [ ] Bug fix
- [x] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other
```

## Objective

Based on SM-417 and [this PR](https://github.com/bitwarden/sdk/pull/30).

This PR adds the create, edit, and delete commands for projects to the `bws` cli.

## Code changes

- **main.rs:** Add CLI functionality for creating, editing, and deleting projects

## Before you submit

- Please add **unit tests** where it makes sense to do so (encouraged but not required)

### CODE REVIEW

File: crates/bws/src/main.rs [hunk 1]
```diff
@@ -90,6 +108,26 @@ enum GetCommand {
...
+#[derive(Subcommand, Debug)]
+enum CreateCommand {
+    Project { name: String },
+}
+
+#[derive(Subcommand, Debug)]
+enum EditCommand {
+    #[clap(group = ArgGroup::new("edit_field").required(true).multiple(true))]
+    Project {
+        project_id: String,
```
Comment (dani-garcia) [inline crates/bws/src/main.rs:120]: If you change this type to be Uuid here directly, you won't need to manually do Uuid::parse_str afterwards

File: crates/bws/src/main.rs [hunk 2]
```diff
@@ -90,6 +108,26 @@ enum GetCommand {
...
+        #[arg(long, group = "edit_field")]
+        name: String,
+    },
+}
+
+#[derive(Subcommand, Debug)]
+enum DeleteCommand {
+    Project { project_ids: Vec<String> },
```
Comment (dani-garcia) [inline crates/bws/src/main.rs:128]: Same for this one

**Review by dani-garcia — COMMENTED** [review]
Looks good, just two small changes

File: crates/bws/src/main.rs (hunk 1 above)
Comment (coltonhurst) [reply crates/bws/src/main.rs]: Great point, updated!

File: crates/bws/src/main.rs (hunk 2 above)
Comment (coltonhurst) [reply crates/bws/src/main.rs]: Great point, updated!

**Review by dani-garcia — APPROVED** [review]
Looks good!

**Pull request merged**